src/BizHawk.Client.Common/movie/bk2/Bk2MnemonicLookup.cs

```diff
@@ -718,12 +718,12 @@ public static string LookupAxis(string button, string systemId)
 				["Motion Left / Right"] = "mX",
 				["Motion Up / Down"] = "mY",
 				["Twist | / |"] = "Twist",
-			},
+			},
 			[VSystemID.Raw.TIC80] = new()
-			{
-				["Mouse Position X"] = "mpX",
-				["Mouse Position Y"] = "mpY",
-				["Mouse Scroll X"] = "msX",
+			{
+				["Mouse Position X"] = "mpX",
+				["Mouse Position Y"] = "mpY",
+				["Mouse Scroll X"] = "msX",
```

Comment:
I don't actually see any changes in this file, is there a spaces vs tabs change or something?